nuxeo-java-client-test/src/test/java/org/nuxeo/client/test/TestRepository.java

```diff
@@ -444,6 +449,68 @@ public void itCanHandleComplexPropertiesWithJson() throws IOException {
         assertEquals("file", document.getTitle());
     }
 
+    /**
+     * Dates can be handled properly if and only if the date is given as {@link java.lang.String}.
+     */
+    @Test
+    public void itCanHandleDateAsString() {
+        GregorianCalendar calendar = new GregorianCalendar(2017, Calendar.MAY, 4, 3, 2, 1);
+        SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSX");
+        calendar.setTimeZone(TimeZone.getTimeZone("UTC"));
+        formatter.setTimeZone(TimeZone.getTimeZone("UTC"));
+        String calendarStr = formatter.format(calendar.getTime());
+        assertEquals("2017-05-04T03:02:01.00Z", calendarStr);
+
+        Document birthday = new Document("Tom's Birthday", "Birthday");
+        birthday.set("dc:title", "Tom's Birthday");
+        birthday.set("bd:name", "Tom");
+        birthday.set("bd:date", calendarStr);
+        birthday = nuxeoClient.repository().createDocumentByPath("/", birthday);
+        assertEquals("Birthday", birthday.getType());
+        assertEquals("Tom", birthday.getPropertyValue("bd:name"));
+        assertEquals("2017-05-04T03:02:01.00Z", birthday.getPropertyValue("bd:date"));
+
+        calendar.add(Calendar.MONTH, 1);
+        calendarStr = formatter.format(calendar.getTime());
+        assertEquals("2017-06-04T03:02:01.00Z", calendarStr);
+        birthday.set("bd:date", calendarStr);
+        birthday = nuxeoClient.repository().updateDocument(birthday);
+        assertEquals("2017-06-04T03:02:01.00Z", birthday.getPropertyValue("bd:date"));
+
+        formatter.setTimeZone(TimeZone.getTimeZone("GMT+1:00"));
+        calendarStr = formatter.format(calendar.getTime());
+        assertEquals("2017-06-04T04:02:01.00+01", calendarStr);
+        birthday.set("bd:date", calendarStr);
+        birthday = nuxeoClient.repository().updateDocument(birthday);
+        assertEquals("2017-06-04T03:02:01.00Z", birthday.getPropertyValue("bd:date"));
```

Comment:
For instance `TimeZone.getTimeZone("Europe/Paris")` or `TimeZone.getTimeZone("GMT+01:00")` are non-UTC timezone. `Z` means UTC. In export formats we normalize timezones to UTC instead of server to be more generic.